class Failure {
  final String message;
  Failure([this.message = "a unexpected error occured"]);
}
